app/views/exports/pbmc_bank_tubes_content_report.csv.erb

```diff
@@ -30,25 +51,13 @@
     # skip if no downstream tubes for this well (not set up children in Limber yet)
     next if source_well.downstream_tubes.empty?
 
-    child_tube_v1 = source_well.downstream_tubes.last
```

Comment:
Was this naming actually not accurate any more? Presumably `@plate.wells_in_columns` used to use the v1 API but now (since Stuart) uses the v2 API?

Reply:
Yes this naming was incorrect, its now of type `Sequencescape::Api::V2::Tube`